Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.h

```diff
@@ -231,6 +231,12 @@ class MediaPlayerPrivateMediaStreamAVFObjC final
 
     MediaStreamTrackPrivate* activeVideoTrack() const;
 
+    // SampleBufferDisplayLayer::Client
+    void sampleBufferDisplayLayerStatusDidFail() final;
+
```

Comment:
I would remove this line

Reply:
The new virtual methods are not declared in SampleBufferDisplayLayer::Client, so I wonder if it makes sense to keep the extra line here. What do you think?

Reply:
Why did you need to move this declaration? Did you get a compile error? If so, I think you should leave your change as-is.